<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualenvConfigurationDialog</class>
 <widget class="QDialog" name="VirtualenvConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Virtualenv Configuration</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Environment Creator</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="virtualenvButton">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Select to use 'virtualenv'</string>
        </property>
        <property name="text">
         <string notr="true">0.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="pyvenvButton">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Select to use 'pyvenv'</string>
        </property>
        <property name="text">
         <string notr="true">0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Paths</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Target Directory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="E5ClearableLineEdit" name="targetDirectoryEdit">
        <property name="toolTip">
         <string>Enter the target directory for the virtual environment</string>
        </property>
        <property name="placeholderText">
         <string>Target directory for the virtual environment</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="targetDirectoryButton">
        <property name="toolTip">
         <string>Select the target directory via a directory selection dialog</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="extraSearchPathLabel">
        <property name="text">
         <string>Extra Search Path:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="E5ClearableLineEdit" name="extraSearchPathEdit">
        <property name="toolTip">
         <string>Enter the extra search path to look for setuptools/pip</string>
        </property>
        <property name="placeholderText">
         <string>Extra search path to look for setuptools/pip</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="extraSearchPathButton">
        <property name="toolTip">
         <string>Select the extra search path to look for setuptools/pip via a directory selection dialog</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="promptPrefixLabel">
        <property name="text">
         <string>Prompt Prefix:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="E5ClearableLineEdit" name="promptPrefixEdit">
        <property name="toolTip">
         <string>Enter the prompt prefix for the virtual environment</string>
        </property>
        <property name="placeholderText">
         <string>Prompt prefix for the virtual environment</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Python Executable:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="E5ClearableLineEdit" name="pythonExecEdit">
        <property name="toolTip">
         <string>Enter the Python interpreter for the virtual environment</string>
        </property>
        <property name="placeholderText">
         <string>Python interpreter for the virtual environment</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="pythonExecButton">
        <property name="toolTip">
         <string>Select the Python interpreter via a file selection dialog</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="verbosityLabel">
          <property name="text">
           <string>Verbosity:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="verbositySpinBox">
          <property name="toolTip">
           <string>Select the verbosity (-1: quiet, 0: normal, 1: verbose)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="versionLabel">
          <property name="text">
           <string>Python Version:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="versionComboBox">
          <property name="toolTip">
           <string>Select the Python version (empty for current)</string>
          </property>
          <item>
           <property name="text">
            <string notr="true"/>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">2.6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">2.7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">3.2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">3.3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">3.4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">3.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.6</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="systemCheckBox">
        <property name="toolTip">
         <string>Select to give the virtualenv access to the global site-packages</string>
        </property>
        <property name="text">
         <string>System-wide Python Packages</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="unzipCheckBox">
        <property name="toolTip">
         <string>Select to unzip setuptools when installing it</string>
        </property>
        <property name="text">
         <string>Unzip Setuptool to virtualenv</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="noSetuptoolsCheckBox">
        <property name="toolTip">
         <string>Select to not install setuptools (or pip) in the new virtualenv</string>
        </property>
        <property name="text">
         <string>Don't install 'setuptool' (or pip) in the virtualenv</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="noPipCcheckBox">
        <property name="toolTip">
         <string>Select to not install pip in the new virtualenv</string>
        </property>
        <property name="text">
         <string>Don't install 'pip' in the virtualenv</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="clearCheckBox">
        <property name="toolTip">
         <string>Select to clear the target first</string>
        </property>
        <property name="text">
         <string>Clear out the target directory</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="copyCheckBox">
        <property name="toolTip">
         <string>Select to always copy files rather than symlinking</string>
        </property>
        <property name="text">
         <string>Always copy files</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="symlinkCheckBox">
        <property name="toolTip">
         <string>Select to use symlinks instead of copies</string>
        </property>
        <property name="text">
         <string>Use Symbolic Links</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="upgradeCheckBox">
        <property name="toolTip">
         <string>Select to upgrade a virtual environment</string>
        </property>
        <property name="text">
         <string>Upgrade</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="logCheckBox">
        <property name="toolTip">
         <string>Select to generate a log file in the target directory</string>
        </property>
        <property name="text">
         <string>Save a log file in the target directory after creation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="scriptCheckBox">
        <property name="toolTip">
         <string>Select to write a shell script/batch file to regenerate the virtualenv</string>
        </property>
        <property name="text">
         <string>Save virtualenv generation script</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="openCheckBox">
        <property name="toolTip">
         <string>Open the newly created virtualenv in a file manager window</string>
        </property>
        <property name="text">
         <string>Open target directory after creation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>E5Gui/E5LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>virtualenvButton</tabstop>
  <tabstop>pyvenvButton</tabstop>
  <tabstop>targetDirectoryEdit</tabstop>
  <tabstop>targetDirectoryButton</tabstop>
  <tabstop>extraSearchPathEdit</tabstop>
  <tabstop>extraSearchPathButton</tabstop>
  <tabstop>promptPrefixEdit</tabstop>
  <tabstop>pythonExecEdit</tabstop>
  <tabstop>pythonExecButton</tabstop>
  <tabstop>verbositySpinBox</tabstop>
  <tabstop>versionComboBox</tabstop>
  <tabstop>systemCheckBox</tabstop>
  <tabstop>unzipCheckBox</tabstop>
  <tabstop>noSetuptoolsCheckBox</tabstop>
  <tabstop>noPipCcheckBox</tabstop>
  <tabstop>clearCheckBox</tabstop>
  <tabstop>copyCheckBox</tabstop>
  <tabstop>symlinkCheckBox</tabstop>
  <tabstop>upgradeCheckBox</tabstop>
  <tabstop>logCheckBox</tabstop>
  <tabstop>scriptCheckBox</tabstop>
  <tabstop>openCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VirtualenvConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VirtualenvConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
